lib/IDE/CodeCompletion.cpp

```diff
@@ -3729,7 +3729,8 @@ class CompletionLookup final : public swift::VisibleDeclConsumer {
         for (auto Ex : NTD->getExtensions()) {
           handleDeclRange(Ex->getMembers(), Reason);
         }
-      } else if (isNameHit(VD->getNameStr())) {
+      } else if (!VD->getBaseName().isSpecial() &&
```

Comment:
It probably *should*.  The situation would be something like
`x[label: #^COMPLETE_HERE^#]`